core/src/main/resources/org/elasticsearch/tasks/task-results-index-mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "result" : {
+    "dynamic" : "strict",
+    "properties" : {
+      "task" : {
+        "properties": {
+          "action": {
+            "type": "keyword"
+          },
+          "cancellable": {
+            "type": "boolean"
+          },
+          "id": {
```

Comment:
`id` is a part of a standard TaskInfo serialization. I was initially thinking about creating a custom serialization of TaskInfo for persistence and removing id from it, but then I thought that it might be useful here since it's a `long` field, which can be useful for sorting since it provides a nice, unambiguous ordering of tasks within a node.